EventTicketingSystem.DataAccess/Models/Settings/DatabaseSettings.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace EventTicketingSystem.DataAccess.Models.Settings
+{
+    /// <summary>
+    /// Settings for Database
+    /// </summary>
+    public class DatabaseSettings
+    {
+        /// <summary>
+        /// Name of section in application settings
+        /// </summary>
+        public const string SectionName = "DatabaseSettings";
+
+        /// <summary>
+        /// Connection string for Database. Contains Host,Port,Database,Username,Password.
+        /// </summary>
+        public string ConnectionString { get; set; }
+
+        /// <summary>
+        /// Connection string template for Database without a password. Contains Host,Port,Database,Username.
+        /// Use '{0}' placeholder for password.
+        /// </summary>
+        public string ConnectionStringTemplate { get; set; }
+
+        /// <summary>
+        /// Password for Username in Connection string for Database.
+        /// </summary>
+        public string ConnectionStringPassword { get; set; }
+
+        /// <summary>
+        /// Method for getting connection string
+        /// </summary>
+        /// <returns>Connection string</returns>
+        public string GetConnectionString()
```

Comment:
looks like property get candidate. Please remove explicit method

Reply:
Good point. But I changed it to simple property and removed unnecessary properties.